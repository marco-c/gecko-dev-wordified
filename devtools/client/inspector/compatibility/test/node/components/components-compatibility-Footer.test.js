/
*
Any
copyright
is
dedicated
to
the
Public
Domain
.
http
:
/
/
creativecommons
.
org
/
publicdomain
/
zero
/
1
.
0
/
*
/
"
use
strict
"
;
/
*
*
*
Unit
tests
for
the
Footer
component
.
*
/
const
{
shallow
}
=
require
(
"
enzyme
"
)
;
const
React
=
require
(
"
react
"
)
;
const
Footer
=
React
.
createFactory
(
require
(
"
devtools
/
client
/
inspector
/
compatibility
/
components
/
Footer
"
)
)
;
describe
(
"
Footer
component
"
(
)
=
>
{
it
(
"
renders
"
(
)
=
>
{
const
pane
=
shallow
(
Footer
(
)
)
;
expect
(
pane
)
.
toMatchSnapshot
(
)
;
}
)
;
}
)
;
