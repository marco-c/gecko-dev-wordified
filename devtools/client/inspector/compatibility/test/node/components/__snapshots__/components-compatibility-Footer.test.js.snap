/
/
Jest
Snapshot
v1
https
:
/
/
goo
.
gl
/
fbAQLP
exports
[
Footer
component
renders
1
]
=
<
footer
className
=
"
compatibility
-
footer
"
>
<
button
className
=
"
compatibility
-
footer__feedback
-
button
"
onClick
=
{
[
Function
]
}
title
=
"
Feedback
"
>
<
img
className
=
"
compatibility
-
footer__feedback
-
icon
"
src
=
"
chrome
:
/
/
devtools
/
skin
/
images
/
report
.
svg
"
/
>
<
label
className
=
"
compatibility
-
footer__feedback
-
label
"
>
Feedback
<
/
label
>
<
/
button
>
<
/
footer
>
;
